Doc (#28431)
huggingface/transformers: docs/source/en/perf_train_cpu.md

| PyTorch Version   | IPEX version   |
| :---------------: | :----------:   |
| 2.1.x             |  2.1.100+cpu   |
| 2.0.x             |  2.0.100+cpu   |
| 1.13              |  1.13.0+cpu    |
| 1.12              |  1.12.300+cpu  |

Please run `pip list | grep torch` to get your `pytorch_version`, so you can get the `IPEX version_name`.
```
pip install intel_extension_for_pytorch==<version_name> -f https://developer.intel.com/ipex-whl-stable-cpu
```

> **___Note: Please check the upgraded versions in [ipex-whl-stable-cpu](https://developer.intel.com/ipex-whl-stable-cpu) if needed.___**

Reviewer: ```suggestion
You can check the latest versions in [ipex-whl-stable-cpu](https://developer.intel.com/ipex-whl-stable-cpu) if needed.
```

--max_seq_length 384 \
--doc_stride 128 \
--output_dir /tmp/debug_squad/ \
<b>--use_ipex</b> \
<b>--bf16</b> \
<b>--use_cpu</b></pre> 

> If you want to enable `use_ipex` and `bf16` in your script, just add some arguments in `TrainingArguments` like this:

Reviewer: Just to be more clear :)

```suggestion
If you want to enable `use_ipex` and `bf16` in your script, add these parameters to `TrainingArguments` like this:
```
